presto-orc/src/main/java/com/facebook/presto/orc/writer/MapFlatColumnWriter.java

```diff
@@ -477,22 +475,14 @@ public KeyManager(T keyToWriter, Supplier<StatisticsBuilder> statisticsBuilderSu
 
         public abstract MapFlatValueWriter getOrCreateValueWriter(int position, Block keyBlock);
 
-        public ColumnStatistics getRowGroupColumnStatistics()
```

Comment:
getRowGroupColumnStatistics are only used for the writer validation, and they actually made writer validation harder.